docs/views.py

```diff
@@ -248,7 +248,9 @@ def upload_to_gisaid(request):
 
 @login_required
 def create_new_user_account(request):
-    converted_to_html = docs.utils.markdown.markdown_to_html("createNewUserAccount.md")
+    converted_to_html = docs.utils.markdown.markdown_to_html(
+        "create_new_user_account.md"
```

Comment:
I think this one should be `"create_new_user.md"` instead of `"create_new_user_account.md"`